docs/working-with-entity-manager.md

```diff
@@ -3,14 +3,14 @@
 Using `EntityManager` you can manage (insert, update, delete, load, etc.) any entity. 
 EntityManager is just like a collection of all entity repositories in a single place.
  
-You can access the entity manager via `getManager()` or from `Connection`.
+You can access the entity manager via DataSource's manager.
 Example how to use it:
  
 ```typescript
-import {getManager} from "typeorm";
+import {Manager} from "./config/DataSource";
```

Comment:
it's unclear what `./config/DataSource` is. To keep it simple I think we should simply show how to intiailize a data source and access it's manager, e.g.:

```ts
const myDataSource = new DataSource({ /*...*/ });
const user = await myDataSource.manager.findOne(User, 1);
user.name = "Umed";
await entityManager.save(user);
```

Reply:
Thank you for your feedback!